### ISSUE TITLE
HTTPException 401 when using api.get_gif()

### ISSUE THREAD

**Issue by jsimons256** [scrazzz/redgifs#10]
### Summary

A couple days ago, the redgifs API started giving a 401 exception when calling api functions.

### Reproduction Steps

```
import redgifs, os
api = redgifs.API()
url = input() # Full url of redgifs video
result = api.get_gif(url.split('/')[-1])
response = api.download(result.urls.hd, os.path.join(path ,dir, name + ".mp4"))
api.close()
```

### Expected Result

Code executes and video is downloaded.

### Actual Result

Traceback (most recent call last):
  File "/path/to/file.py", line 26, in <module>
    result = api.get_gif(sub.url.split('/')[-1])
  File "/path/to/site-packages/redgifs/api.py", line 92, in get_gif
    json: Dict[str, Any] = self.http.get_gif(id)['gif']
  File "/path/to/site-packages/redgifs/http.py", line 123, in get_gif
    return self.request(r, **params)
  File "/path/to/site-packages/redgifs/http.py", line 111, in request
    raise HTTPException(r, js)
redgifs.errors.HTTPException: 401 (Error: {'code': 'MissingAuthorizationHeader', 'description': 'authorization required to access this service', 'docs': 'https://github.com/Redgifs/api/wiki/API-access'})

### System Information

- Python v3.9.14-final
- redgifs v1.3.1-final
- aiohttp v3.8.2
- system info: Darwin 21.6.0 Darwin Kernel Version 21.6.0: Wed Aug 10 14:25:27 PDT 2022; root:xnu-8020.141.5~2/RELEASE_X86_64

### Checklist

- [X] I have searched all open and closed issues for any duplicates.

### Additional Information

The code used to work a couple days ago, but suddenly stopped working.

**Comment by scrazzz** [comment]
Thanks for opening a bug report.

Yep, this is RedGIFs making breaking changes to their API. Looks like this endpoint now requires an API key which you have 0.1% of getting it (see [this](https://github.com/scrazzz/redgifs/issues/4#issuecomment-1276255509) to know what I'm talking about).

I'm a bit busy right now IRL so I'll see what I can do about this later.

**Comment by scrazzz** [comment]
I've fixed this issue on 1.4.0 (08e42b638bc79ad46b2ef9108fbed003c3ca6a45) with a temporary login method. You can check the example in the README or in the examples folder on how to use it.
